acceptance-tests/postgresql_test.go

```diff
@@ -107,3 +73,58 @@ var _ = Describe("PostgreSQL", func() {
 		Expect(uriIP.IsPrivate()).To(BeFalse())
 	})
 })
+
+func postgresTestMultipleApps(serviceInstance *services.ServiceInstance) {
+	By("pushing the unstarted app twice")
```

Comment:
praise: thank you very much for the abstraction
You may want to add the `GinkgoHelper()` to ensure the location presented to the user will always be in the spec that called the helper, and not the helper itself

Reply:
Thanks for pointing that out. Added.